src/dev.c

```diff
@@ -195,6 +198,18 @@ fido_init(int flags)
 {
 	if (flags & FIDO_DEBUG || getenv("FIDO_DEBUG") != NULL)
 		fido_log_init();
+#ifdef USE_HIDAPI
+	if (hid_init() != 0)
+		errx(1, "hid_init failed");
```

Comment:
Having the library call errx() and exit the application is not ideal. If you are worried about hid_init() failing, then I suggest keeping a static variable in hid_hidapi.c to track whether hid_init() has been called, and returning an error from there.

Reply:
According to hidapi.h's documentation hid_init and hid_exit calls are not necessary unless multi-threading is involved: https://github.com/signal11/hidapi/blob/master/hidapi/hidapi.h#L82.
Thus I removed these calls altogether, a quick test of examples/cred on my Linux workstation at least works.